docker/docker-compose.yml

```diff
@@ -115,8 +115,29 @@ services:
       - mailhog
       - minio
 
+  # To transition from pg12 to pg13, run ./bin/upgrade.pg12.to.pg13
+  pg12:
+    build:
+      context: docker/pg12
+    container_name: pg12
+    environment:
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_USER: postgres
+      POSTGRES_MULTIPLE_EXTENSIONS: postgis,hstore
+      PGPASSWORD: postgres
+      PGUSER: postgres
+    volumes:
+      - dbdata:/var/lib/postgresql/data
+    ports:
+      - 5432
+    expose:
+      - 5432
+
+  # This should no longer need to be overridden for ARM
+  # processors
   db:
-    image: postgis/postgis:13-3.3-alpine
+    build:
+      context: docker/pg13
```

Comment:
What re the chances we can make this postgis.  That's the main reason I have an override (I think) `image: gangstead/postgis:13-3.1-arm`

Reply:
The Dockerfile installs postgis, and I confirmed that postgis works for me locally.